neurodocker/__init__.py

```diff
@@ -1,20 +1,19 @@
-import sys
+"""Neurodocker generates containers for neuroimaging.
+
+It includes ReproEnv, which is an extensible, generic container generator.
+"""
+
+from pathlib import Path
 
 from neurodocker._version import get_versions
+from neurodocker import reproenv  # noqa: F401
 
 __version__ = get_versions()["version"]
 del get_versions
 
-from neurodocker.generators import Dockerfile
-from neurodocker.generators import SingularityRecipe
-from neurodocker.utils import set_log_level
-
-
-if sys.version_info[0] < 3:
-    raise RuntimeError(
-        "Neurodocker requires Python 3. Use Neurodocker's Docker image or"
-        " install Python 3."
-    )
-
+# Register neurodocker templates
+# TODO: remove registration from the `generate` cli. otherwise we register twice.
```

Comment:
is this TODO addressed? If not perhaps an issue can be created to remember it.

Reply:
I think this TODO is addressed. The only place templates are registered in the CLI is below. And that should probably stay as it is, because it allows the user to specify alternative template paths. https://github.com/ReproNim/neurodocker/blob/106539f7fc28cf5f6c1a9a8f71404599f9b07874/neurodocker/cli/generate.py#L54-L62